CIP-0049/README.md

```diff
@@ -1,24 +1,27 @@
 ---
 CIP: 49
 Title: ECDSA and Schnorr signatures in Plutus Core
-Authors: Koz Ross (koz@mlabs.city), Michael Peyton-Jones (michael.peyton-jones@iohk.io), Iñigo Querejeta Azurmendi (querejeta.azurmendi@iohk.io)
-Discussions-To: koz@mlabs.city
 Status: Proposed
```

Comment:
Has this been included in Plutus core? I thought it had

Reply:
Included in the valentine hardfork https://docs.cardano.org/cardano-testnet/about/secp/

Reply:
OK, will make it so; stay tuned for commit(s).  Last they told us, it was still hung up on costing (that's all that had been recorded in the PR discussion)... so will tick that box as well since it never would have been mainstreams without a full costing analysis.

Reply:
resolved with https://github.com/cardano-foundation/CIPs/pull/700/commits/34da9ac518f3a28d4ae20077c3c126b3fb248108